app/src/main/java/com/nextcloud/talk/webrtc/PeerConnectionWrapper.java

```diff
@@ -535,10 +535,8 @@ public void onRemoveStream(MediaStream mediaStream) {
 
         @Override
         public void onDataChannel(DataChannel dataChannel) {
-            if ("status".equals(dataChannel.label()) || "JanusDataChannel".equals(dataChannel.label())) {
-                PeerConnectionWrapper.this.dataChannel = dataChannel;
-                PeerConnectionWrapper.this.dataChannel.registerObserver(new MagicDataChannelObserver());
-            }
+            PeerConnectionWrapper.this.dataChannel = dataChannel;
```

Comment:
Similar to https://github.com/nextcloud/talk-ios/pull/1557#discussion_r1504056575 it probably would make sense to log something if a `dataChannel` has already been stored.